docker-compose.yml

```diff
@@ -77,7 +77,7 @@ services:
   #   Database Service
   #-----------------------------------------------
   db:
-    image: postgres:10.6-alpine
+    image: postgres:11.1-alpine
```

Comment:
was this necessary? will it break databases from 10.6?

Reply:
```
2019-01-28 23:33:08.684 UTC [1] FATAL:  database files are incompatible with server 
2019-01-28 23:33:08.684 UTC [1] DETAIL:  The data directory was initialized by PostgreSQL version 11, which is not compatible with this version 10.6.
```
Error I get if I try and use 10.6. Not totally sure why? something to do with my local postgres?

Reply:
You mean uninstall postgres and re-install an older version? doesn't that kind of defeat the purpose of docker? plus that will break anything else I'm using postgres for locally? Or am i misunderstanding you?

Reply:
I think there's a miscommunication; we can drop this version down, merge this, and fix your machine tomorrow